spec.bs

```diff
@@ -5733,7 +5932,7 @@ An <dfn>auction config</dfn> is a [=struct=] with the following [=struct/items=]
   :: An [=origin=].
     The origin of the seller running the ad auction. The [=origin/scheme=] must be "`https`".
   : <dfn>decision logic url</dfn>
-  :: A [=URL=].
+  :: Null or a [=URL=].

```

Comment:
Add a short sentence to explain when it can be null, and when is it required (not nullable)?

Reply:
done